api/envoy/config/route/v3/route_components.proto

```diff
@@ -1474,6 +1488,8 @@ message RateLimit {
 
       // Rate limit on the existence of request headers.
       HeaderValueMatch header_value_match = 6;
+
+      DynamicMetaData dynamic_metadata = 7;
```

Comment:
Can you add a one liner comment above as per other `oneof` alternatives?

Reply:
Done